src/App.css

```diff
@@ -1,38 +1,43 @@
+body {
+    width: 50vw;
+    margin: 0 auto;
+}
+
```

Comment:
I would have tried to apply this to a styled component. See if  you can research using a container and using max-width to simplify this.